transactions.sql

```diff
@@ -0,0 +1,31 @@
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species='pokemon' WHERE name NOT LIKE '%mon%';
```

Comment:
For this query we are looking for where name ends with `mon` and where name does not end with `mon`, what you have will target names with `mon` in between

- [ ] Please update query